bytes_ostream.hh

```diff
@@ -268,6 +268,35 @@ public:
         write(bytes_view(reinterpret_cast<const signed char*>(ptr), size));
     }
 
+    // Writes the fragmented view
```

Comment:
> bytes_ostream: overload write() to support more input types

Can narrow down to FragmentedView, now you're not supporting various integer types.